CHANGELOG.md

```diff
@@ -6,42 +6,45 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## \[Unreleased]
+
 ### Added
 
-- \[SDK\] A `DeferredTqdmProgressReporter` class, which doesn't have glitchy output
-  like `TqdmProgressReporter` in certain circumstances
+- \[SDK\] Introduced the `DeferredTqdmProgressReporter` class,
+  which avoids the glitchy output seen with the `TqdmProgressReporter` under certain circumstances
   (<https://github.com/opencv/cvat/pull/6556>)
-- \[SDK, CLI\] A `cvat_sdk.auto_annotation` module that provides
-  functionality for automatically annotating a task by running a
-  user-provided function on the local machine, and a corresponding CLI command
-  (`auto-annotate`)
-  (<https://github.com/opencv/cvat/pull/6483>,
-  <https://github.com/opencv/cvat/pull/6649>)
-- Cached frames indication on the interface (<https://github.com/opencv/cvat/pull/6586>)
+- \[SDK, CLI\] Added predefined functions using torchvision
+  and the `cvat_sdk.auto_annotation` module,
```

Comment:
The module needs to be described first, because it provides context for the predefined functions. Something like "Added the `cvat_sdk.auto_annotation` module, providing functionality to [...]. Some predefined functions using torchvision are also available."